requirements.txt

```diff
@@ -3,3 +3,6 @@ sphinx-cjkspace
 sphinx-copybutton
 sphinx-panels
 sphinx_rtd_theme
+ipykernel
```

Comment:
This has to be installed, otherwise, an error will be reported. See https://nbsphinx.readthedocs.io/en/0.8.5/installation.html#Jupyter-Kernel.